project/plugins.sbt

```diff
@@ -4,16 +4,16 @@ resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases
 
 resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.15.0")
```

Comment:
Can you update the resolvers here please? There's a link on the cerb tech channel that Rob posted explaining the change